### PR TITLE
Tests all remaining request classes

### PR DESCRIPTION
**Pull request by pouladpld**
Following issue #611

### CODE REVIEW

**Comment by pouladpld** [comment]
I keep this PR open for a while until tests are done. Meanwhile, feel free to review them.

File: test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        public GetUserProfileTests(TestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [OrderedFact(DisplayName = FactTitles.ShouldLeaveChat)]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.LeaveChat)]
+        public async Task Should_Get_User_Profile_Photos()
+        {
+            // ToDo: Exception when leaving private chat
```
Comment (Tuscen) [inline test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: todo from another test

File: test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                userId: _fixture.BotUser.Id
+            );
+            
+            Assert.True(1 <= profilePhotos.TotalCount);
+            Assert.NotNull(profilePhotos.Photos.First());
+        }
+
+        private static class FactTitles
+        {
+            public const string ShouldLeaveChat = "Should leave chat";
```
Comment (Tuscen) [inline test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: Fact from another test

File: test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs (hunk 1 above)
Comment (pouladpld) [reply test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: copy pasta. right? :D

Comment (Tuscen) [reply test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: Everybody do it sometimes :D

File: test/Telegram.Bot.Tests.Integ/Other/LeaveChatTests.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        private readonly TestsFixture _fixture;
+
+        public LeaveChatTests(TestsFixture fixture, Fixture classFixture)
+        {
+            _fixture = fixture;
+            _classFixture = classFixture;
+        }
+
+        [OrderedFact(DisplayName = FactTitles.ShouldLeaveChat,
+            Skip = "Bot should stay in chat for other the test cases")]
```
Comment (Tuscen) [inline test/Telegram.Bot.Tests.Integ/Other/LeaveChatTests.cs]: Bot can ask the user to create a group and add it to the group

Comment (pouladpld) [reply test/Telegram.Bot.Tests.Integ/Other/LeaveChatTests.cs]: that requires too much interaction by user. I was trying to keep it as automated as possible. It's a very simple method though.

File: test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs (hunk 1 above)
Comment (pouladpld) [reply test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: fixed

File: test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs (hunk 2 above)
Comment (pouladpld) [reply test/Telegram.Bot.Tests.Integ/Other/GetUserProfileTests.cs]: fixed

File: docs/wikis/tests/sys-integ-tests.md [hunk 4]
```diff
@@ -148,3 +148,7 @@ If the following 3 settings are not set, bot will ask a tester to send it `/me`
 ```json
 { "RegularGroupMemberId": 1234567890 }
 ```
+
+#### Games
+
+Bot should have a game with short name: `game1`.
```
Comment (karb0f0s) [inline docs/wikis/tests/sys-integ-tests.md]: + Bot should have Inline Feedback set to 100% with @BotFather

File: test/Telegram.Bot.Tests.Integ/Games/GamesExceptionTests.cs [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+                    gameShortName: "non_existing_game"
+                )
+            );
+
+            Assert.Equal("game_short_name", e.Parameter);
+        }
+
+        private static class FactTitles
+        {
+            public const string ShouldThrowInvalidGameShortNameException = "Should throw InvalidGameShortNameException";
```
Comment (karb0f0s) [inline test/Telegram.Bot.Tests.Integ/Games/GamesExceptionTests.cs]: add new line here?

File: test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs [hunk 6]
```diff
@@ -68,11 +108,114 @@ public GamesTests(TestsFixture fixture, GamesFixture classFixture)
...
+
+            Assert.Equal(MessageType.Game, gameMessage.Type);
+        }
+
+        [OrderedFact(DisplayName = FactTitles.ShouldAnswerGameCallbackQuery)]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.AnswerCallbackQuery)]
+        public async Task Should_Answer_CallbackQuery_With_Game_Url()
+        {
+            await _fixture.SendTestCaseNotificationAsync(FactTitles.ShouldAnswerGameCallbackQuery,
+                "Click on any Play button on game messges above 👆");
```
Comment (karb0f0s) [inline test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs]: on game messAges above

File: test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs [hunk 7]
```diff
@@ -27,6 +27,8 @@ public GamesTests(TestsFixture fixture, GamesFixture classFixture)
             _classFixture = classFixture;
         }
 
+        #region Regualr Game Message
```
Comment (karb0f0s) [inline test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs]: Regular

File: docs/wikis/tests/sys-integ-tests.md (hunk 4 above)
Comment (pouladpld) [reply docs/wikis/tests/sys-integ-tests.md]: Yes. That should be added to wiki.

File: test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs (hunk 6 above)
Comment (pouladpld) [reply test/Telegram.Bot.Tests.Integ/Games/GamesTests.cs]: _any game messages above_

**Pull request merged**